source/draw/images/DefaultFont.ooc

```diff
@@ -0,0 +1,206 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+// Original typeface for printing debug messages that was crudely drawn by David Piuva in a few minutes because it was quicker than understanding the license terms of existing fonts.
+
+GlobalCleanup register(|| DefaultFontAsciiImage free())
+DefaultFontAsciiImage := "
```

Comment:
The style is a bit inconsistent because only `I` and `J` uses serifs. `I` has to differ from `l`, `J` has to be consistent with `I` and characters have fixed width rendering so that it can later be GPU optimized.